src/Model/Scope/Condition.php

```diff
@@ -104,7 +104,7 @@ public function __construct($key, $operator = null, $value = null)
             throw new Exception('Only Scope can contain another conditions');
         } elseif ($key instanceof Field) { // for BC
             $key = $key->short_name;
-        } elseif (!is_string($key) && !($key instanceof Expression)) {
+        } elseif (!is_string($key) && !($key instanceof Expression) && !($key instanceof Expressionable)) {
```

Comment:
probably yes, but why `Expression` does not implement `Expressionable` (and does not even provide that one prototyped method)?

where is `Expressionable` used in Scopes?

Reply:
$key can be Expressionable.
Expressionable is any object that can be converted to expression using getSqlExpression method